scripts/protein/esm2/test_pydantic_train.py

```diff
@@ -0,0 +1,136 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-Apache2
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shlex
+import sqlite3
+import subprocess
+from pathlib import Path
+
+import pandas as pd
+import pytest
+from lightning.fabric.plugins.environments.lightning import find_free_network_port
+
+from bionemo.testing.data.load import load
+
+
+data_path: Path = load("single_cell/testdata-20240506") / "cellxgene_2023-12-15_small" / "processed_data"
+
+
+def test_bionemo2_rootdir():
+    data_error_str = (
+        "Please download test data with:\n"
+        "`python scripts/download_artifacts.py --models all --model_dir ./models --data all --data_dir ./ --verbose --source pbss`"
+    )
+    assert data_path.exists(), f"Could not find test data directory.\n{data_error_str}"
+    assert data_path.is_dir(), f"Test data directory is supposed to be a directory.\n{data_error_str}"
+
+
+@pytest.fixture
+def dummy_protein_dataset(tmp_path):
```

Comment:
Can be imported from [here](https://github.com/NVIDIA/bionemo-framework/blob/f5e096cd05397547f2398858045fbd70b1ef69e1/sub-packages/bionemo-testing/src/bionemo/testing/data/esm2.py)